app/graphql/types/query_type.rb

```diff
@@ -59,6 +59,8 @@ def record_id(id:, index:)
                                             'you with one for your specific use case'
 
       argument :source, String, required: false, default_value: 'All', deprecation_reason: 'Use `sourceFilter`'
+      argument :boolean_type, String, required: false, default_value: 'OR',
+                                      description: 'How to join multiword queries. Defaults to "OR" which means any of the words much match. Options include: "OR", "AND", "minimum_a"'
```

Comment:
Yeah so these docs are already out of date with the code. Will fix before merge.

Reply:
Ha, you beat me to it. Ignore my comment below.